scripts/typedoc.sh

```
@@ -0,0 +1,27 @@
+do
+  if [ -d $directory ]; then
+    bn=`basename $directory`;
+    echo "typedoc compiling $directory"
+    pnpx $TYPEDOC --version
+    pnpx $TYPEDOC --json ${OUTPUT_DIR}/$bn.json --tsconfig $directory/tsconfig.json $directory/src/index.ts;
+    SUCCESS+=("${OUTPUT_DIR}/$bn.json")
+  fi
+done
+
+echo "============"
+for f in ${SUCCESS[@]}; do
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)